cli/app.go

```diff
@@ -1051,11 +1091,22 @@ var app = &cli.App{
 											Name:  trainFlagModelVersion,
 											Usage: "version of ML model. defaults to current timestamp if unspecified.",
 										},
+										&cli.StringFlag{
+											Name:     mlTrainingFlagURL,
+											Usage:    "url of Github repository associated with the training scripts",
+											Required: false,
+										},
 										&cli.StringFlag{
 											Name:     mlTrainingFlagPath,
 											Usage:    "path to ML training scripts for upload",
 											Required: true,
 										},
+										// CR erodkin: flag this for etai/tahiya, make sure they're happy with the changes!
+										&cli.StringFlag{
+											Name:     generalFlagOrgID,
+											Usage:    "org ID to train and save ML model in",
+											Required: true,
+										},
```

Comment:
(flyby) added this flag, which was being searched for in the `withUpload` method but wasn't settable.
@etai-shuchatowitz @tahiyasalam I copied the flag details from other commands, if you have any issues with the shape of the flag here (usage text, etc.), please let me know!

Reply:
Ah looking through this now, I am realizing there's actually a bug in our existing `withUpload` method.

The `trainFlagModelOrgID` [here](https://github.com/viamrobotics/rdk/blob/9bcc9592ba1a6f5efe96dd425337d43b715fe65c/cli/ml_training.go#L55) and [here](https://github.com/tahiyasalam/rdk/blob/9bcc9592ba1a6f5efe96dd425337d43b715fe65c/cli/ml_training.go#L61) should actually in fact be the `generalFlagOrgID`, where the `generalFlagOrgID` is the "org ID to save the custom training script in". I hate to sneak in too many changes here, so I'm also happy with removing this from this PR & taking on as a follow up for my team or merging in that fix first for you to pull down into your branch. Let me know!

Reply:
Hey, I think that's not a problem to do here! I just made the change, let me know if it still looks off :)

Reply:
Looks good to me! :)